src/isar/storage/storage_service.py

```diff
@@ -40,10 +40,7 @@ def store(
 
         self.storage.store(data=result.data, path=destination_path)
 
-    def store_metadata(
-        self,
-        mission: Mission,
-    ) -> None:
+    def store_metadata(self, mission: Mission, inspections: List[Inspection]) -> None:
```

Comment:
Take in the mission id and the mission metadata instead of taking in the whole mission?